java-tests/validating/src/test/kotlin/io/spine/test/options/setonce/SetOnceIntegerITest.kt

```diff
@@ -47,19 +47,19 @@ internal class SetOnceIntegerITest {
     @Nested inner class
     `prohibit overriding non-default 'int32'` {
 
-        private val currentBalance = balance { uSD = SIXTY }
-        private val newBalance = balance { uSD = SIXTEEN }
+        private val currentBalance = studentSetOnce { cashUSD = SIXTY }
```

Comment:
To avoid any clashes and ease reading, I would use the values which names differ **more** than "sixty" differs from "sixteen" on reading.